### ISSUE TITLE
SceneViewer doesn't work on chrome with android

### ISSUE THREAD

**Issue by christianpugger** [google/model-viewer#1286]
AR doesn't work on chrome neither with android, just me?

if I insert the code on wordpress I have this problem, even on the html page, while on Wix it works, how come?

**Comment by elalish** [comment]
Good question; can you post links here where it does and doesn't work? I'd be happy to take a look. Also, does the javascript console show any interesting errors or warnings? A possible issue is not having the proper CORS headers on the glTF.

**Comment by FrankHiguera** [comment]
Hello, I wanted to share my experience and I think we have the same symptom.

I have tested the following two links with model-viewer using WebXR and SceneViwer, I have also downloaded the same demos from GITHUB and uploading them to my server the same thing happens (The launch AR button dont start the AR).

Links:

>> (WEBXR) https://modelviewer.dev/examples/webxr.html
>> (WEBXR + SCENEVIEWER) https://modelviewer.dev/examples/augmented-reality.html

Using model-viewer: 
>> version 1.0

Browsers version and device:

>>> Samsung browser (11.2.2.3 GalaxyS7 and 11.2.2.3 GalaxyS10) in both case happen the same:
WebXR - AR button dont start the AR.
Sceneview - AR button dont start the AR.

>>> Google Chrome browser (83.0.4 GalaxyS7 and 83.0.41 GalaxyS10) in both case happen the same:
WebXR - This is the ONLY case where the button start the AR.
Sceneview - AR button dont start the AR.

This start since model-viewer version 1.0, it was working in 0.10
